src/auth/authClient.tsx

```diff
@@ -60,11 +66,23 @@ const refresh: (refreshToken: string) => Promise<RefreshTokenResponse> = (
     },
   }).then((response) => response.data);
 
+// TODO: maybe make these functions return Promise<Boolean> to let the user know what's up
+const forgotPassword: (user: ForgotPasswordRequest) => Promise<void> = (
+  user: ForgotPasswordRequest,
+) => AuthAxiosInstance.post(API_ROUTE.FORGOT_PASSWORD, user);
```

Comment:
Do either of these routes return any data? Doesn't seem like it... 

You can chain .then.catch onto these functions _where you call them_ to alert the user.